src/app/exam/score/ExamScorePage.js

```diff
@@ -7,15 +7,26 @@ import {useExamContext} from "../questions/ExamContext";
 import {Spinner} from "../../commons/Spinner";
 import {TableCell} from "../../../utils/TableCell";
 import ExamSummary from "./ExamSummary";
-import {examTranscriptService} from "../../../services/services";
+import {examService, examTranscriptService} from "../../../services/services";
 import {ExamScorePresenter} from "./ExamScorePresenter";
+import {ExportCsvButton} from "../../commons/buttons/ExportCsvButton";
 
 
 const ExamScorePage = () => {
     const {url: currentURL} = useRouteMatch();
     const {examId} = useParams();
     const {refetchExam, currentExam} = useExamContext();
     const [presenter, setPresenter] = useState(undefined);
+    const onButtonExportCSVClick = (examId) => {
+        examService.getExamTranscriptCsvFile(examId)
+        .then((res) => {
+            const url = window.URL.createObjectURL(new Blob([res.data]));
+            const link = document.createElement('a');
+            link.href = url;
+            link.setAttribute('download', 'ExamTranscript.csv');
+            link.click();
+        });
+    };
```

Comment:
We will move this function below the `useEffect(...)`(s). Function(s) not used by any `useEffect(...)` is prefered to be put after `useEffect(...)`(s).